gradle/gradle-mvn-push.gradle

```diff
@@ -95,6 +95,10 @@ afterEvaluate {
                             name = 'Olivier Perez'
                             email = 'olivier@olivierperez.fr'
                         }
+                        developer {
```

Comment:
@MiSikora Found out that we had you missing for quite some time here in POM. Decided to fix it here. Will do a PR for `develop` as well later.
Let me know if this data is fine (took it from one of your repos 😄 ) and if you would like to add your email as well here.

Reply:
Yup, looks good. Skipping email is fine :).